serveradmin/serverdb/models.py

```diff
@@ -600,6 +603,34 @@ class Meta:
         index_together = [['attribute', 'value']]
 
 
+class ServerDateTimeAttribute(ServerAttribute):
+    attribute = models.ForeignKey(
+        Attribute,
+        db_index=False,
+        on_delete=models.CASCADE,
+        limit_choices_to=dict(type='datetime'),
+    )
+    value = models.DateTimeField()
+
+    class Meta:
+        app_label = 'serverdb'
+        db_table = 'server_datetime_attribute'
+        unique_together = [['server', 'attribute', 'value']]
+        index_together = [['attribute', 'value']]
+
+    def save(self, *args, **kwargs):
+        # Our transport datetime format doesn't include a timezone.  Django
```

Comment:
I couldn't get why.  As far as I understand, Django gives timezone aware datetimes.

Reply:
It's more about what we give django. Our transport format will be displayed pretty much 1:1 in servershell so it should be humanly accessible first. I asked @TuxPowered42 if we want timezones and or millisecond precision there and he said no to both.

Reply:
Please check again :)